### ISSUE TITLE
alt text same

### ISSUE THREAD

**Issue by discospiff** [nityak04/IS7024#13]
Should the alt text be the same on each of these?  The images all have different names, which indicates they should be different.

https://github.com/nityak04/IS7024/blob/1f757498065ae1aef122b79a53c3703a8e96c36c/TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml#L25
